lib/v2/polkadot/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/polkadot/blog': ['iceqing'],
```

Comment:
The route path should be the same as `router.js`
```suggestion
    '/blog': ['iceqing'],
```